bindings.c

```diff
@@ -654,7 +651,7 @@ bool cgfs_remove(const char *controller, const char *cg)
 	if (fd < 0)
```

Comment:
Oh, I guess so.

Could you also make sure that when recursive_rmdir() fails, fd gets closed, and when the fdopendir there fails, dupfd gets closed?